monai/bundle/scripts.py

```diff
@@ -1212,14 +1216,8 @@ def _export(
     # add .json extension to all extra files which are always encoded as JSON
     extra_files = {k + ".json": v for k, v in extra_files.items()}
 
-    save_net_with_metadata(
-        jit_obj=net,
-        filename_prefix_or_stream=filepath,
-        include_config_vals=False,
-        append_timestamp=False,
-        meta_values=parser.get().pop("_meta_", None),
-        more_extra_files=extra_files,
-    )
+    saver(net, filepath, more_extra_files=extra_files)
```

Comment:
When calling `saver`, you could include the argument `meta_values=parser.get().pop("_meta_", None)` so that this line isn't needed in the partial constructs you've added, which could then fit onto one line. You would have to state that `saver` expects the `meta_values` argument in the doctring above. You can leave this unchanged if you want, only a suggestion.

Reply:
Thank you for your suggestion! This idea will be discussed along with another proposed change in the comments below.